### PR TITLE
WebTestVerifyText

### PR DESCRIPTION
**Pull request by JuliaChernakov**

### CODE REVIEW

File: src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package qa_java_beginners;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class JuliaChernakovHW12Test {
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java:10]: You can extend from BaseTest as you did in Song99BottlesJuliaChernakovTest.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java (hunk 1 above)
Comment (JuliaChernakov) [reply src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java:10]: Thank you very much, I'll try to fix it

File: src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java [hunk 2]
```diff
@@ -16,28 +14,16 @@ public class JuliaChernakovHW12Test extends BaseTest {
...
+        getDriver().findElement(By.xpath("//li/a[@href='/abc.html']")).click();
+        getDriver().findElement(By.xpath("//a[@href='j.html']")).click();
+        WebElement text = getDriver().findElement(By.xpath("//p[strong]"));
         String actualResult = text.getText();
 
         Assert.assertEquals(actualResult, expectedResult);
-
-        driver.quit();
     }
 }
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java:29]: You do not need this empty line as well.

File: src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java [hunk 3]
```diff
@@ -16,28 +14,16 @@ public class JuliaChernakovHW12Test extends BaseTest {
...
 
-        WebElement menuBrowseLanguages = driver.findElement(By.xpath("//li/a[@href='/abc.html']"));
-        menuBrowseLanguages.click();
-
-        WebElement submenuJ = driver.findElement(By.xpath("//a[@href='j.html']"));
-        submenuJ.click();
+        String expectedResult = "All languages starting with the letter J are shown, sorted by Language.";
 
-        WebElement text = driver.findElement(By.xpath("//p[strong]"));
+        getDriver().get("http://www.99-bottles-of-beer.net");
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/JuliaChernakovHW12Test.java:20]: You might add space after this line.

**Review by anonymoustuna — APPROVED** [review]

**Pull request merged**